api_tests/nodes/views/test_view_only_query_parameter.py

```diff
@@ -403,7 +412,7 @@ def test_node_list_view_only_links(
         assert assertions != 0
```

Comment:
This might be a good place to verify that the only thing returned is a type.

Reply:
implemented similar test on UserDetail view - this is for contributors, so bibliographic and permission are also returned, but links, id, should be stripped.